File: src/main/resources/webapp/angular-book/src/app/book/book.ts
export class Book {
  id: number;
  title: string;
  author: string;
  coverPhotoUrl: string;
  isbnNumber: number;
  price: number;
  language: string;

  constructor(id: number, title: string, author: string, coverPhotoUrl: string, isbnNumber: number, price: number, language: string) {
    this.id = id;
    this.title = title;
    this.author = author;
    this.coverPhotoUrl = coverPhotoUrl;
    this.isbnNumber = isbnNumber;
    this.price = price;
    this.language = language;
  }

}

File: src/main/java/com/book/bookrestapi/entity/Book.java
package com.book.bookrestapi.entity;

import com.sun.istack.NotNull;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;

@Data
@Entity
@ToString
@AllArgsConstructor
@NoArgsConstructor
public class Book {

    @Id
    @GeneratedValue
    private Long id;

    @NotNull
    private String title;

    @NotNull
    private String author;

    @NotNull
    private String coverPhotoUrl;

    @NotNull
    private Long isbnNumber;

    @NotNull
    private Double price;

    @NotNull
    private String language;
}
